java-squid/src/test/java/org/sonar/java/se/ExplodedGraphWalkerTest.java

```diff
@@ -98,9 +98,10 @@ public void scanFile(JavaFileScannerContext context) {
     }
 
     private void reportIssuesFor(JavaFileScannerContext context, JavaCheck check) {
-      Multimap<Tree, String> issues = ((DefaultJavaFileScannerContext) context).getSEIssues((Class<? extends SECheck>) check.getClass());
-      for (Map.Entry<Tree, String> issue : issues.entries()) {
-        context.reportIssue(check, issue.getKey(), issue.getValue());
+      Multimap<Tree, DefaultJavaFileScannerContext.SEIssue> issues = ((DefaultJavaFileScannerContext) context).getSEIssues((Class<? extends SECheck>) check.getClass());
+      for (Map.Entry<Tree, DefaultJavaFileScannerContext.SEIssue> issue : issues.entries()) {
+        DefaultJavaFileScannerContext.SEIssue seIssue = issue.getValue();
+        context.reportIssue(this, seIssue.getTree(), seIssue.getMessage(), seIssue.getSecondary(), null);
```

Comment:
The test fails to show what the end user will see on the screen. So, the effect of the change remains quite theoretical.